databases/backends/postgres.py

```diff
@@ -10,11 +10,20 @@
 from sqlalchemy.sql.schema import Column
 from sqlalchemy.types import TypeEngine
 
+from databases.backends.common import ConstructDefaultParamsMixin
 from databases.core import LOG_EXTRA, DatabaseURL
 from databases.interfaces import ConnectionBackend, DatabaseBackend, TransactionBackend
 
 logger = logging.getLogger("databases")
 
+_result_processors = {}  # type: dict
+
+
+class APGCompiler_psycopg2(
```

Comment:
Seems strange to call this `APGCompiler_psycopg2`, given that the db backend used for postgres in databases is `asyncpg`, not `psycopg2`.

Reply:
this is how aiopg calls it
https://github.com/encode/databases/blob/master/databases/backends/aiopg.py#L32#L35

Reply:
Right but that's because aiopg is just a wrapper around psycopg2, asyncpg is not.